CONTRIBUTING.md

```diff
@@ -21,51 +21,6 @@ or that there are particular issues that you should know about before implementi
 You will have to fork the `apm-server` repo,
 please follow the instructions in the [readme](README.md).
 
```

Comment:
removing this section creates a gab in the contributing document, it is true that the info in the template, but if I read the contributing document there are some steps that I will only see when the PR is ready.

```suggestion
  [Checks before submitting your changes](.github/pull_request_template.md)
```

Reply:
I like this approach! @simitt @axw wdyt?

Reply:
What about:
```markdown
### Submitting your changes

Please read our [pull request template](.github/pull_request_template.md), which includes the information we care the most when submitting new changes.
```